modules/ui/src/main/scala/lucuma/ui/primereact/primereact.scala

```diff
@@ -11,5 +11,13 @@ import reactST.StBuildingComponent
 import scalajs.js
 
 extension (button: Button)
-  def compact     = button.copy(clazz = button.clazz.toOption.orEmpty |+| LucumaStyles.Compact)
-  def veryCompact = button.copy(clazz = button.clazz.toOption.orEmpty |+| LucumaStyles.VeryCompact)
```

Comment:
It turns out that although the `VeryCompact` Css is being used on a few buttons in Explore at the moment, due to precedence issues, it really isn't changing anything from `compact`. If we want more compact buttons, we can revisit this.